core/src/test/java/edu/wpi/grip/core/metrics/BenchmarkRunnerTest.java

```diff
@@ -0,0 +1,99 @@
+package edu.wpi.grip.core.metrics;
+
+import edu.wpi.grip.core.events.BenchmarkEvent;
+import edu.wpi.grip.core.events.RunStartedEvent;
+import edu.wpi.grip.core.events.RunStoppedEvent;
+
+import com.google.common.eventbus.EventBus;
+import com.google.common.eventbus.Subscribe;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Tests for {@link BenchmarkRunner}.
+ */
+public class BenchmarkRunnerTest {
+
+  private EventBus eventBus;
+  private BenchmarkRunner benchmarkRunner;
+
+  @Before
```

Comment:
Why is this even getting reported??? This is the _only_ public test method getting this; another with _exactly the same signature_ isn't getting flagged for this. Nor is this breaking the build (as it should if checkstyle doesn't pass, so I'm guessing codacy is just broken)